pkg/plugins/plugins.go

```diff
@@ -148,13 +148,17 @@ func Get(name, imp string) (*Plugin, error) {
 	if err != nil {
 		return nil, err
 	}
-	for _, driver := range pl.Manifest.Implements {
-		logrus.Debugf("%s implements: %s", name, driver)
-		if driver == imp {
-			return pl, nil
+	if imp != "" {
```

Comment:
So `Implements` is an array.  Basic pattern here to iterate and return if found otherwise error after.  How early are you thinking?  Initial `get` is the cache of the driver.